encoding/serialization.go

```diff
@@ -75,6 +94,10 @@ func (c *G2Commitment) UnmarshalJSON(data []byte) error {
 	}
 	c.X = g2Point.X
 	c.Y = g2Point.Y
+
```

Comment:
Same here, let's remove this function if unused.

Reply:
dataapi?